metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/service/DatasetService.java

```diff
@@ -193,6 +194,7 @@ public void updateDataset(MetisUser metisUser, Dataset dataset, String xsltStrin
     if (xsltString == null) {
       dataset.setXsltId(storedDataset.getXsltId());
     } else {
+      cleanDatasetXslt(storedDataset.getXsltId());
```

Comment:
Is it your intention to do a separate full database cleanup? If not, I am worried that a lot of historic records are never cleaned up because it only looks at the current xslt reference (that it is replacing) instead of all the ones that came before as well. What is your plan for those?

Reply:
Yes the idea is that we should update those scripts to include the cleanup of xslts as well. The data for those are generally not that big or crucial, so we don't really need a daemon running every day. Even if we forget cleaning one time the data will be very minimal. Nonetheless we could include a cleanup during a reindexing operation or even an "extra" daemon in metis-core that could be performed monthly or so..